cross/fossil/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = fossil-scm
+PKG_VERS = 2.10
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.fossil-scm.org/index.html/tarball/version-$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-v$(PKG_VERS)
+
+BUILD_DEPENDS = cross/zlib
+BUILD_DEPENDS += cross/openssl
+
+HOMEPAGE = http://www.fossil-scm.org/
+COMMENT  = Fossil is a simple, high-reliability, distributed software configuration management system.
+LICENSE  = http://www.fossil-scm.org/home/doc/trunk/COPYRIGHT-BSD2.txt
```

Comment:
for well known licenses place the type here: `LICENSE = BSD2`
And use the same in the spk Makefile